### ISSUE TITLE
Windows nightly latest?

### ISSUE THREAD

**Issue by fcore117** [qTox/qTox#5600]
Please fix Win builds so it has dark theme included.

**Comment by anthonybilinski** [comment]
Confirmed, the nightly's hash as of today is d2508e7eac26d731c746333b5d74e9a051d3f5b7 which is from the end of January.

@nurupo @sudden6, any idea what's going wrong with the build server? Sudden mentioned he restarted the build but that maybe the source isn't being correctly updated.

**Comment by arnysch** [comment]
Could it be that a nightly build of Jenkins' project [qtox_src](https://build.tox.chat/view/qtox/job/qtox_src/) should be triggered somehow automatically when there is a commit in qtox's master branch?

The different real qtox nightly builds seem to depend on qtox_src.

**Comment by nurupo** [comment]
A web hook setup in qTox repo should notify Jenkins of new pushes and update qtox_src. So either someone deleted the hook, or the hook stopped working for some reason, or Jenkins can't process the hook for some reason, etc. Can someone with access to qTox repo settings check if there is a hook setup for Jenkins, what is the URL, when was the last notification sent and what was Jenkin's reply to it?

**Comment by nurupo** [comment]
Also, I should really hurry with my Python GitHub nightly publishing script, which would allow to move qTox nighties to Travis... Anyway, let's toubleshoot the issue with Jenkins for now.

**Comment by nurupo** [comment]
So, I just created a test GitHub repo and a test job on Jenkins. I have setup the webhook on GitHub to notify Jenkins of pushes and I have setup the Jenkins job similarly to `qotx_src` job but with my repo's URL. Everything works just fine. https://build.tox.chat/job/nurupo-test1_src/

**Comment by arnysch** [comment]
Some info is [here](https://build.tox.chat/view/qtox/job/qtox_src/lastBuild/pollingLog/):

> Started on Jan 29, 2019 9:06:54 PM
Started by event from 192.30.252.35 ⇒ https://build.tox.chat:8080/github-webhook/ on Tue Jan 29 21:06:54 UTC 2019
Using strategy: Default
[poll] Last Built Revision: Revision 013771c13a9e68a97d113cf268e52a4751cb4505 (refs/remotes/origin/master)
 > git --version # timeout=10
 > git ls-remote -h https://github.com/qTox/qTox # timeout=10
Found 16 remote heads on https://github.com/qTox/qTox
[poll] Latest remote head revision on refs/heads/master is: d2508e7eac26d731c746333b5d74e9a051d3f5b7
Done. Took 0.8 sec
Changes found

Of course does not show why it is no longer working

**Comment by nurupo** [comment]
Here is how it's setup

![jenkins_hook](https://user-images.githubusercontent.com/1855294/55296820-de641900-53ec-11e9-91e4-8e3a8bed47a9.png)

**Comment by anthonybilinski** [comment]
Thanks for looking into it. I took a look at our audit log and it looks like I changed some kind of hook config on Jan 24th - I'm not sure if that was related to something like CI but it's suspiciously close to when nightlies stopped, and also github services which we had a jenkins plugin for that hits the build.tox.chat URL are now deprecated. We didn't have any current webhooks for build.tox.chat but did have that jenkins service that doesn't look like it's running. I've now added a push webhook to the build.tox.chat URL like you showed in your screenshot, so the server will hopefully be hit on next push to the repo.
